### PR TITLE
delete function with new testing & style cleanup

### PR DESCRIPTION
**Pull request by gurmankalkat**

### CODE REVIEW

File: src/test/java/seedu/meal360/Meal360Test.java [hunk 1]
```diff
@@ -5,14 +5,14 @@
 import java.util.HashMap;
 import org.junit.jupiter.api.Test;
 
+
 class Meal360Test {
 
     @Test
     public void sampleTest() {
         assertTrue(true);
     }
```
Comment (JuneNita) [inline src/test/java/seedu/meal360/Meal360Test.java:15]:
```
@test
```
i think you accidentally deleted this at line 15 :)

File: src/test/java/seedu/meal360/Meal360Test.java [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
 
-    @Test
     public void testViewRecipe() {
```
Comment (JuneNita) [inline src/test/java/seedu/meal360/Meal360Test.java:16]:
```
@test
```

i think you accidentally deleted this line :)

File: src/main/java/seedu/meal360/Meal360.java [hunk 3]
```diff
@@ -29,6 +29,12 @@ public static void receiveInput(String input) {
         String[] command = input.trim().split(" ");
         if (input.equalsIgnoreCase("bye")) {
             canExit = true;
+        // delete a recipe in list
+        } else if (command[0].equals("delete")) {
+            Recipe deletedRecipe = parser.parseDeleteRecipe(command, recipeList);
```
Comment (TopGun2001) [inline src/main/java/seedu/meal360/Meal360.java:34]: Suggestion to add exception handling for all commands in future.

File: src/main/java/seedu/meal360/Meal360.java [hunk 4]
```diff
@@ -29,6 +29,12 @@ public static void receiveInput(String input) {
...
+            ui.printMessage("Noted. I've removed this recipe:");
+            ui.printMessage(deletedRecipe.toString());
+            ui.printMessage("Now you have " + recipeList.size() + " recipes in the list.");
         } else if (command[0].equals("view")) {
             Recipe recipe = parser.parseViewRecipe(command, recipeList);
```
Comment (TopGun2001) [inline src/main/java/seedu/meal360/Meal360.java:39]: Same as above, need to add exception.

**Pull request merged**